specifications/scroll.json

```
@@ -679,6 +679,34 @@
           "url": "https://eips.ethereum.org/EIPS/eip-2935"
         }
       }
+    },
+    {
+      "metadata": {
+        "name": "galileo",
+        "block": 1765868400
```

```suggestion
        "block": 26639740
```

Use block number instead of timestamp